src/Gui/SoFCDB.cpp

```diff
@@ -74,6 +74,10 @@
 #include "Inventor/SmSwitchboard.h"
 #include "Inventor/SoAutoZoomTranslation.h"
 #include "Inventor/SoDrawingGrid.h"
+#include "SoFCBackgroundGradient.h"
+#include "SoFCBoundingBox.h"
+#include "SoMouseWheelEvent.h"
+#include "SoFCTransform.h"

```

Comment:
SoFCTransform is still inside Inventor dir and the rest are already included further up 
```suggestion
#include "Inventor/SoFCTransform.h"
```